src/mono/wasm/runtime/marshal-to-cs.ts

```diff
@@ -49,6 +49,66 @@ export function initialize_marshalers_to_cs(): void {
     }
 }
 
+export function bind_arg_marshal_to_cs(sig: JSMarshalerType, marshaler_type: MarshalerType, index: number): Function {
+
+    if (marshaler_type === MarshalerType.None || marshaler_type === MarshalerType.Void) {
+        return () => undefined;
+    }
+    let res_marshaler: MarshalerToCs | undefined = undefined;
+    let arg1_marshaler: MarshalerToJs | undefined = undefined;
+    let arg2_marshaler: MarshalerToJs | undefined = undefined;
+    let arg3_marshaler: MarshalerToJs | undefined = undefined;
+
+    const marshaler_type_res = get_signature_res_type(sig);
+    if (marshaler_type_res !== MarshalerType.None) {
+        const converter = js_to_cs_marshalers.get(marshaler_type_res);
+        mono_assert(converter && typeof converter === "function", () => `Unknow converter for type ${marshaler_type_res} at ${index}`);
+
+
+        if (marshaler_type != MarshalerType.Nullable) {
+            res_marshaler = converter;
+        }
+        else {
+            marshaler_type = marshaler_type_res;
+        }
+    }
+
+    const marshaler_type_arg1 = get_signature_arg1_type(sig);
+    if (marshaler_type_arg1 !== MarshalerType.None) {
+        const converter = cs_to_js_marshalers.get(marshaler_type_arg1);
+        mono_assert(converter && typeof converter === "function", () => `Unknow converter for type ${marshaler_type_arg1} at ${index}`);
```

Comment:
```suggestion
        mono_assert(converter && typeof converter === "function", () => `Unknown converter '${converter}' for type ${marshaler_type_arg1} at ${index}`);
```
should this have one of our error codes with a number? I'm guessing it's more an internal failure than something we expect users to see